Dockerfile.cucumber-build

```diff
@@ -3,7 +3,7 @@
 #
 FROM alpine:3.7
 
-WORKDIR /app
+WORKDIR /go/src/github.com/cucumber/cucumber
```

Comment:
This wasn't necessary for building `gherkin/go`. Is it really necessary for `tag-expressions/go`? I'd like to avoid too many language-isms in this Docker image. 

It's supposed to be a kitchen sink with all language dependencies, but without favouring any particular one.